source/administration/data-retention.rst

```diff
@@ -5,28 +5,28 @@ Available in `Enterprise Edition E20 <https://about.mattermost.com/pricing/>`__.
 
 By default, Mattermost stores all message history providing an unlimited search history to admins and end users.
 
-In Enterprise Edition E20, you may set a custom policy for how long messages and file uploads are kept in Mattermost channels and direct messages.
+In Enterprise Edition E20, you can set a custom policy to manage how long messages and file uploads are kept in Mattermost channels and direct messages.
 
 .. warning:: Once a message or a file is deleted, the action is irreversible. Please be careful when setting up a custom data retention policy.
 
 .. toctree::
     :maxdepth: 2
 
-Set Up Guide
---------------
+Configuring a Data Retention Policy
+------------------------------------
 
 To set a custom data retention policy:
 
-1. Go to **System Console > Advanced > Data Retention Policy** in prior versions or **System Console > Compliance > Data Retention Policy** in versions after 5.12.
-2. Set your desired retention policy for messages. Messages, including file attachments, older than the duration you set will be deleted nightly. The minimum message retention time is one day.
-3. Similarly, set your desired retention policy for file uploads. File uploads older than the duration you set will be deleted nightly from your file storage system, either from your local disk or your Amazon S3 service as specified in **System Console > Files > Storage** in prior versions or **System Console > Environment > File Storage** in versions after 5.12. The minimum file retention time is one day.
+1. Go to **System Console > Compliance > Data Retention Policy** (or **System Console > Advanced > Data Retention Policy** in versions prior to 5.12).
+2. Select a **Message Retention** option. Messages, including file attachments, older than the duration you set will be deleted nightly. The minimum message retention time is one day.
+3. Select a **File Retention** option. File uploads older than the duration you set will be deleted nightly from your file storage system, either from your local disk or your Amazon S3 service as specified in **System Console > Environment > File Storage** (or **System Console > Files > Storage** in versions prior to 5.12). The minimum file retention time is one day.
 4. Set the start time of the daily scheduled data retention job. Choose a time when fewer people are using your system. Must be a 24-hour time stamp in the form HH:MM.
 
-Save the settings and restart your server. You're now all set! Data older than the duration you set are now deleted daily at the specified server time.
+Save the settings and restart your server. Messages and files older than the duration you set will be deleted daily at the specified server time.
```

Comment:
Again, deleted daily vs. a specified time, coupled with the "data deletion time" is confusing.